### PR TITLE
Finish digits, persian_content, numeric_suffixes and national_code modules

### PR DESCRIPTION
**Pull request by omid**

### CODE REVIEW

File: src/digit/mod.rs [hunk 1]
```diff
@@ -21,27 +26,55 @@ fn digits_convert<T: AsRef<str>>(src: T, from: Lang, to: Lang) -> String {
...
 
+/// Takes a string which may contain Arabic digits, and returns
+/// a string that represents the same digits but in Persian.
+pub fn digits_ar_to_fa<T: AsRef<str>>(src: T) -> String {
+    digits_convert(src, Lang::Ar, Lang::Fa)
+}
+
+/// Takes a string which may contain Arabic digits, and returns
+/// a string that represents the same digits but in English.
+pub fn digits_ar_to_en<T: AsRef<str>>(src: T) -> String {
```
Comment (robatipoor) [inline src/digit/mod.rs:49]: I think it is a better idea to find the type of language in our function from the input content with regex

File: src/sheba/mod.rs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+pub fn verify_card_number<T: AsRef<str>>(number: T) -> bool {
+    let number = number.as_ref();
+    let len = number.len();
+    if len < 16
+        || number.chars().any(|c| !c.is_numeric())
```
Comment (robatipoor) [inline src/sheba/mod.rs:5]: i think use `number.parse::<i32>().is_err()` better performance

**Review by robatipoor — COMMENTED** [review]
thank you omid

File: src/sheba/mod.rs (hunk 2 above)
Comment (omid) [reply src/sheba/mod.rs:5]: what??? it shouldn't be here :D it's absolutely incomplete!

File: src/digit/mod.rs (hunk 1 above)
Comment (omid) [reply src/digit/mod.rs:49]: What if it contains both Arabic and English?

Comment (robatipoor) [reply src/digit/mod.rs:49]: We give him an error

Comment (omid) [reply src/digit/mod.rs:49]: But it's not an error :D

**Pull request merged**